Source/WebCore/platform/PublicSuffixStore.h

```diff
@@ -38,14 +38,28 @@ class PublicSuffixStore {
 
     // https://url.spec.whatwg.org/#host-public-suffix
     WEBCORE_EXPORT bool isPublicSuffix(StringView domain) const;
-    WEBCORE_EXPORT String publicSuffix(const URL&) const;
     WEBCORE_EXPORT String topPrivatelyControlledDomain(const String& host) const;
     WEBCORE_EXPORT void clearHostTopPrivatelyControlledDomainCache();
+    WEBCORE_EXPORT void enablePublicSuffixCache();
 
+    class PublicSuffix {
```

Comment:
Maybe we can define this *outside* of the PublicSuffixStore scope? I don't feel strongly but it feels like this might be a bit nicer.

Reply:
Odds are we may want to add it to its own header too, with its own HashTraits and isolatedCopy().